tests/robustness/traffic/traffic.go

```diff
@@ -149,13 +149,120 @@ func SimulateTraffic(ctx context.Context, t *testing.T, lg *zap.Logger, clus *e2
 	lg.Info("Reporting traffic during failure injection", zap.Int("successes", duringFailpointStats.Successes), zap.Int("failures", duringFailpointStats.Failures), zap.Float64("successRate", duringFailpointStats.SuccessRate()), zap.Duration("period", duringFailpointStats.Period), zap.Float64("qps", duringFailpointStats.QPS()))
 	lg.Info("Reporting traffic after failure injection", zap.Int("successes", afterFailpointStats.Successes), zap.Int("failures", afterFailpointStats.Failures), zap.Float64("successRate", afterFailpointStats.SuccessRate()), zap.Duration("period", afterFailpointStats.Period), zap.Float64("qps", afterFailpointStats.QPS()))
 
+	watchTotal := CalculateWatchStats(reports, startTime, endTime)
+	watchBefore := CalculateWatchStats(reports, startTime, fr.Start)
+	watchDuring := CalculateWatchStats(reports, fr.Start, fr.End)
+	watchAfter := CalculateWatchStats(reports, fr.End, endTime)
```

Comment:
I don't think there is much information from splitting stats for a streaming operation like watch for the substages. This makes sense for unary KV operation that are short, as there is small chance that they will spread between multiple stages.

I would start from just collecting  and reporting `watchTotal` stats.